docker/addons/bootstrap/docker-compose.grpc-mtls.yaml

```
@@ -0,0 +1,14 @@
+services:
+  bootstrap:
+    environment:
+      # Users gRPC client environmental varaibles
+      MF_AUTH_GRPC_CLIENT_MTLS: ${MF_USERS_GRPC_CLIENT_MTLS}
+      MF_AUTH_GRPC_CLIENT_TLS: ${MF_USERS_GRPC_CLIENT_TLS}
+      MF_AUTH_GRPC_CLIENT_CERT: /users-grpc-client.crt
```

```suggestion
      MF_AUTH_GRPC_CLIENT_CERT: ${MF_AUTH_GRPC_CLIENT_CERT}
```

Then we update `.env` file same way we do with vault token

---

@rodneyosodo .  The `MF_AUTH_GRPC_CLIENT_CERT` variables are shared with both addons and mainflux core service , 
For mainflux core service path will be `./ssl/certs` for addons path will be `../../ssl/certs` 
So only added `ADDON_CERT_PREFIX_PATH` variable for addon services